client/state/selectors/is-nav-unification-enabled.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal dependencies
+ */
+import { isAutomatticTeamMember } from 'calypso/reader/lib/teams';
+import { getReaderTeams } from 'calypso/state/teams/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
```

Comment:
If we remove `import QueryReaderTeams from 'calypso/components/data/query-reader-teams';` we should query the `ReaderTeams` in this component.

Reply:
Yeah, that's what I tried to say in https://github.com/Automattic/wp-calypso/pull/50711#discussion_r586402078. Unfortunately we cannot use `QueryReaderTeams` here because this is not a component.